base/stream.jl

```diff
@@ -876,6 +894,14 @@ function stop_reading(stream::LibuvStream)
     end
 end
 
+function readbytes!(s::LibuvStream, b::AbstractArray{UInt8}, nb=length(b))
```

Comment:
i think they differ a bit on what happens with the stream is closed before the array is fully populated (not that i think we shouldn't change that)